tempodb/encoding/vparquet4/testqt/floatattr.txt

```diff
@@ -0,0 +1,54 @@
+rebatchIterator: 
```

Comment:
should we remove these?

Reply:
It seems it's not so easy to run benchmark tests. Need time to figure out how to deal with this. Looks like testblock should be somehow generated. :)
```go
	r, _, _, err := local.New(&local.Config{
		// Path: path.Join("/Users/marty/src/tmp"),
		// Path: path.Join("/Users/mapno/workspace/testblock"),
		Path: path.Join("/Users/joe/testblock"),
	})
	require.NoError(b, err)
```